examples/call.rs

```diff
@@ -165,8 +165,8 @@ valkey_module! {
     allocator: (ValkeyAlloc, ValkeyAlloc),
     data_types: [],
     commands: [
-        ["call.test", call_test, "", 0, 0, 0],
-        ["call.blocking", call_blocking, "", 0, 0, 0],
-        ["call.blocking_from_detached_ctx", call_blocking_from_detach_ctx, "", 0, 0, 0],
+        ["call.test", call_test, "", "", 0, 0, 0],
+        ["call.blocking", call_blocking, "", "", 0, 0, 0],
+        ["call.blocking_from_detached_ctx", call_blocking_from_detach_ctx, "", "", 0, 0, 0],
     ],
```

Comment:
if you run ./test.sh you get an error
```
failures:
---- test_call stdout ----
Error: failed to start valkey server
Caused by:
    0: Loading valkey module: ...valkeymodule-rs/target/debug/examples/libcall.dylib
    1: No such file or directory (os error 2)
failures:
    test_call
    test_call_blocking
```
for some reason call.rs example doesn't compile.  I can't figure out why.  Can you repro this?